app/controllers/concerns/curation_concerns/collections_controller_behavior.rb

```diff
@@ -15,6 +15,9 @@ module CollectionsControllerBehavior
 
       # This is needed as of BL 3.7
       copy_blacklight_config_from(::CatalogController)
+      configure_blacklight do |config|
+        config.search_builder_class = CurationConcerns::CollectionSearchBuilder
```

Comment:
Then it was never necessary since that was the only consumer of the module it moved from?